lms/djangoapps/course_api/views.py

```diff
@@ -158,11 +162,13 @@ class CourseListView(APIView):
             ]
     """
 
-    def get(self, request):
+    pagination_class = NamespacedPageNumberPagination
+    serializer_class = CourseSerializer
+    paginate_by = 20
```

Comment:
Oh right.  But do we want to set that parameter site-wide?

I think the precedence order is (highest to lowest):

1. GET /url?{page_size_query_param}=[x]
2. View.paginate_by = [y]
3. settings.PAGINATE_BY = [z]